.vscode/settings.json

```diff
@@ -21,5 +21,7 @@
       "url": "https://cdn.jsdelivr.net/npm/tsup/schema.json",
       "fileMatch": ["package.json", "tsup.config.json"]
     }
-  ]
+  ],
+  "typescript.tsdk": "node_modules/typescript/lib",
```

Comment:
we don't have Typescript at root, so i think this wouldn't take effect.

Reply:
🤦‍♂️ Yup.